models/chainlink/CCIP/chainlink_CCIP_events.sql

```diff
@@ -0,0 +1,122 @@
+{{ config(tags=['chainlink', 'CCIP'],
```

Comment:
Hi @Synthquest, if these spells are dune sql spells, you may need to add `dunesql` into tags.

```suggestion
{{ config(tags=['chainlink', 'CCIP', 'dunesql'],
```

Reply:
Here is the guide you can follow:
https://github.com/duneanalytics/spellbook#how-can-i-contribute-spells-on-dunesql

Reply:
All dune sql spells in this PR may need the `dunesql` tag.